src/inverseRobot.js

```diff
@@ -7,7 +7,16 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const revers = {};
+
+  for (const key in robot) {
+    if (robot[key] in revers) {
```

Comment:
create variable for `robot[key]`, because u use it twice and its a improve readability
also `in` can work not correctly, try `.hasOwnProperty`